fineract-provider/build.gradle

```diff
@@ -382,12 +381,10 @@ tasks.withType(JavaCompile) {
                 "ProtectedMembersInFinalClass",
                 "StaticAssignmentInConstructor",
                 "UnescapedEntity",
-                "EmptyCatch",
```

Comment:
Wait, what? No, this one I want... :smile: This is related to FINERACT-942 and https://checkstyle.sourceforge.io/config_blocks.html#EmptyCatchBlock @thesmallstar we should be able to enforce this by moving to error, agreed?

Reply:
Yep will move it